udata_front/theme/gouvfr/templates/organization/list.html

```diff
@@ -23,12 +24,35 @@
 {% endblock %}
 
 {% block main_content %}
-<section class="container">
-    <div class="pt-sm search-input">
-        <form method="GET" class="row-inline fs-lg">
-            <span class="icon active">{% include theme("svg/search.svg") %}</span>
-            <input type="text" name="q" value="{{organizations.query._query}}" aria-label="{{ _('Search...') }}"
-                placeholder="{{ _('Search...') }}" />
+<section class="fr-container fr-py-2w">
+    <h1 class="fr-mb-1w">{{_('Organizations')}}</h1>
+    <div>{{_('Search an organization on %(site)s', site=current_site.title)}}</div>
+    <div class="fr-pt-3v">
+        <form method="GET" class="fr-griw-row">
+            <div class="fr-search-bar w-100" role="search">
+                <label class="fr-label" for="search-organizations">
+                    {{ _('Search...') }}
+                </label>
+                <input
+                    name="q"
+                    class="fr-input"
+                    placeholder="{{ _('Search...') }}"
+                    type="search"
+                    id="search-organizations"
+                    value="{{organizations.query._query}}"
+                />
+                <button class="fr-btn" title="{{_('Search')}}" type="submit">
+                    {{ _('Search') }}
+                </button>
+            </div>
+            <div class="w-100 fr-grid-row fr-mt-7v fr-pt-3v justify-between align-items-center fr-grid-row--middle border-top border-default-grey">
+                <p class="fr-m-0">{{ _('%(result)s results', result=organizations.total) }}</p>
+                {{sort_search(organizations,
+                created=(_('Creation date'), 'desc'),
```

Comment:
Maybe we can defined `sorts` at the top for more visibility? (similar to dataset/list.html)

Reply:
Done with the same format everywhere.